yarn.lock

```diff
@@ -2814,8 +2814,7 @@
     remark-slug "^6.0.0"
     ts-dedent "^2.0.0"
     util-deprecate "^1.0.2"
-
-"@storybook/addon-essentials@^6.4.22":
+  "@storybook/addon-essentials@^6.4.22":
```

Comment:
Can we remove these changes? Looks like only formatting was changed.

Reply:
Fixed.